src/features/movies/MovieList/movieListSaga.js

```diff
@@ -11,7 +11,7 @@ function* fetchMovieListHandler() {
         const fetchedMovies = yield call(getMovieList)
         yield put(setMovieList(fetchedMovies))
     } catch (error) {
-        yield call(fetchMovieListError())
+        yield put(fetchMovieListError())
         yield call(alert, 'cos nie tak z listy')
```

Comment:
I suggest writing error/alert info in English so the browser is dedicated to everyone.